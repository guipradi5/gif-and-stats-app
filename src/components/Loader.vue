<template>
  <div class="ui-loader">
      <img src="@/assets/svg/loader.svg"/>
  </div>
</template>

<script>
export default {
  name: "Loader"
}
</script>

<style scoped lang="scss">
.ui-loader{
  display:block;
  text-align: center;
}
</style>