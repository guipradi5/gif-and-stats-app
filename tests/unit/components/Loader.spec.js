import { shallowMount } from "@vue/test-utils";
import Loader from "@/components/Loader.vue";

const component = () => {
  return shallowMount(Loader, {})
}

describe("Loader.vue", () => {
  it('Loader should mount', () => {
    const wrapper = component()
    expect(wrapper.vm).toBeTruthy()
  })

  it('Loader should render', async () => {
    const wrapper = await component()
    expect(wrapper.html()).toMatchSnapshot()
  })
});
