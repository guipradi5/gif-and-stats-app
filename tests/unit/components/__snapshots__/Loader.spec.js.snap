// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue Loader should render 1`] = `<div class="ui-loader"><img src=""></div>`;
